### PR TITLE
Dashboard design

### PR DESCRIPTION
**Pull request by Tsz-Fung-Ryan**
implemented dashboard to hide/display website routes from main page

### CODE REVIEW

**Comment by Tsz-Fung-Ryan** [comment]
resolved merge conflicts

File: src/app/app.component.ts [hunk 1]
```diff
@@ -7,4 +7,10 @@ import { Component } from '@angular/core';
 })
 export class AppComponent {
   title = 'angular-dnd-campaign-manager';
+  sideBarShown = false;
+
+
+  toggleSidebar(){
```
Comment (thomas-brunton) [inline src/app/app.component.ts:13]: Please add a unit test, thank you.

**Review by thomas-brunton — CHANGES_REQUESTED** [review]
Please add a unit test for toggling the sidebar. Very important.

File: src/app/app.component.ts (hunk 1 above)
Comment (Tsz-Fung-Ryan) [reply src/app/app.component.ts:13]: unit test added in latest commit

**Review by thomas-brunton — COMMENTED** [review]
Looks good

**Pull request merged**